package.json

```diff
@@ -14,6 +14,7 @@
   },
   "scripts": {
     "dist": "webpack && webpack --config=webpack.config.minified.js",
+    "lib": "rm -rf ./lib && babel ./modules -d lib && rm -rf ./lib/__tests__ ./lib/__mocks__",
```

Comment:
Also, as `GLOBAL` comment, remove all `./` prefixes as Unix/Mac doesn't need them and they break Windows.